db/migrate/20170425154145_add_sti_to_middlewares.rb

```diff
@@ -0,0 +1,89 @@
+class AddStiToMiddlewares < ActiveRecord::Migration[5.0]
+  class MiddlewareDatasource < ApplicationRecord
+    self.inheritance_column = :_type_disabled
```

Comment:
I thought the patch was adding STI but I am sure I'm missing something :confused: